stardis/base.py

```diff
@@ -134,6 +145,9 @@ class STARDISOutput:
         Numpy array of wavelengths used for spectrum with units of Angstroms.
     """
 
+    ###TODO: Instead of returning all these various quantities of the radiation, simply return
+    # the radiation field with class properties that return useful quantities such as spectrum lambda and lambdas.
```

Comment:
spectrum lambda and lambdas?

Reply:
Those are just the names that were used for F_lambda at the outermost point and the wavelength array. I'm open to changing those names as well, but are just the ones that were already implemented and not specifically the focus of this PR.